Jenkinsfile

```
@@ -57,21 +57,34 @@ pipeline {
             if (env.BRANCH_NAME == cambpmDefaultBranch()) {
               // CE master triggers EE master
               // otherwise CE PR branch triggers EE PR branch
-              eeBranch = "cambpm-ee-main/pipeline-master"
+              eeMainProjectBranch = "cambpm-ee-main/pipeline-master"
```

We should fix this. There is no `pipeline-master` branch anymore.
```suggestion
              eeMainProjectBranch = "cambpm-ee-main/master"
```

---

This was mentioned today by @yanavasileva ... as I understand it's not there but could be created anytime if needed till we merge everything.

---

Then, I'm approving this PR. :smile: